src/ppom-controller.ts

```diff
@@ -246,12 +248,12 @@ export class PPOMController extends BaseControllerV2<
     blockaidPublicKey,
   }: {
     chainId: string;
-    onNetworkChange: (callback: (networkState: any) => void) => void;
+    onNetworkChange: (callback: (networkState: never) => void) => void;
```

Comment:
why is this `never` and not `NetworkState`?

Reply:
In general, `(...args: `SuperType`) => T` is a *subtype* of `(...args: `SubType`) => T`, so typing function arguments as `never` gives us the widest function supertype for the given return type. 

We could definitely use `NetworkState` here, but that will narrow the type, which could be a breaking change.

Reply:
Hmm it doesn't break any tests here at least. Updated `never` to `NetworkState`: https://github.com/MetaMask/ppom-validator/pull/89/commits/14cfb3b90121589720db1c8583908ad0ebc780bf